lms/urls.py

```diff
@@ -609,9 +609,13 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
-    urlpatterns += static(
-        settings.PROFILE_IMAGE_DOMAIN + settings.PROFILE_IMAGE_URL_PATH, document_root=settings.MEDIA_ROOT
-    )
+    # DO NOT MERGE this configuration should not be conditional, even in devstack/DEBUG,
+    # but it's here for the moment as it permits validating using the s3 backend in devstack.
+    if 'location' in settings.PROFILE_IMAGE_BACKEND_CONFIG['storage_kwargs']:
```

Comment:
it's the opposite, we should serve images via django in case of DEBUG.  the intention is to get rid of the special case before we merge (or at least, not have the condition depend on such poorly-encapsulated config details).